src/SASLib.jl

```diff
@@ -2,7 +2,7 @@ __precompile__()
 
 module SASLib
 
-using StringEncodings, Missings
+using StringEncodings, Missings, Dates
```

Comment:
This should be ``Base.Dates`` to import the ``Dates`` module that ships with julia.